xbmc/windowing/wayland/ShellSurfaceWebOSShell.cpp

```diff
@@ -0,0 +1,138 @@
+/*
+ *  Copyright (C) 2017-2018 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#include "ShellSurfaceWebOSShell.h"
+
+#include "Registry.h"
+
+#include "utils/log.h"
+
+using namespace KODI::WINDOWING::WAYLAND;
+using namespace std::placeholders;
+
+using namespace wayland;
+
+/*
+  WebOS always is 1920x1080
+*/
+#define WEBOS_UI_WIDTH 1920
```

Comment:
Please put these in an anonymous namespace with constexpr.
```suggestion
namespace
{

constexpr int WEBOS_UI_WIDTH = 1920
constexpr int WEBOS_UI_HEIGHT = 1080

}
```